index.html

```diff
@@ -11,8 +11,8 @@ <h1 id="title">Dr. Norman Borlaug</h1>
         <img id="image" src="https://upload.wikimedia.org/wikipedia/commons/4/43/Norman_Borlaug_2.jpg" alt="Dr. Norman Borlaug">
         <p id="img-caption">Dr. Norman Borlaug, father of the Green Revolution</p>
       </div>
-      <p id="tribute-info">Dr. Norman Borlaug was an American agronomist and humanitarian who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution. Borlaug was awarded multiple honors for his contributions to agriculture, including the Nobel Peace Prize, the Presidential Medal of Freedom, and the Congressional Gold Medal.</p>
-      <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Learn more about Dr. Norman Borlaug</a>
+      <p id="tridbute-info">Dr. Norman Borlaug was an American agronomist and humanitarian who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution. Borlaug was awarded multiple honors for his contributions to agriculture, including the Nobel Peace Prize, the Presidential Medal of Freedom, and the Congressional Gold Medal.</p>
+      <a id="tridbute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Learn more about Dr. Norman Borlaug</a>
```

Comment:
- Though everything is good in your code but there is small typo . i suggest you to change 'tridbute' to 'tribute'.